code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -44,6 +44,8 @@
 					oldmag.update_icon()
 				else
 					to_chat(user, "<span class='notice'>You insert the magazine into \the [src].</span>")
+
+				playsound(user, 'sound/weapons/autoguninsert.ogg', 60, 1)
```

Comment:
I have a issue here where the sound only plays when someone performs a 'tac reload' and not placing the magazine when the gun has no mag present.